src/Aquifer.API/Endpoints/Notifications/List/Endpoint.cs

```diff
@@ -151,82 +161,101 @@ private static async Task<IReadOnlyList<ResourceCommentNotificationData>> GetRes
         DbConnection dbConnection,
         int userId,
         DateTime minimumCommentCreatedDate,
+        IConfiguration cofiguration,
         CancellationToken ct)
     {
+        var userIds = GetNotifyUserIdsOnCommunityReviewerComment(cofiguration);
```

Comment:
It's a long name but It's easier to read the code.  I also suggest passing this list into this method instead of `configuration`.
```suggestion
        var communityReviewerCommentNotificationUserIds = GetNotifyUserIdsOnCommunityReviewerComment(configuration);
```